packages/preview/funarray-0.2.0/typst.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "funarray"
+version = "0.2.0"
+entrypoint = "funarray.typ"
+authors = ["Ludwig Austermann"]
+license = "MIT"
+description = "A package providing convinient functional functions for typst to use on arrays."
+repository = "https://ludwig-austermann/typst-funarray"
```

Comment:
Link not working, probably you forgot `github.com/` ?

Reply:
Yes, you are right, thanks. The urls should now be fixed for both packages.